astro.config.ts

```diff
@@ -54,7 +54,6 @@ export default defineConfig({
   output: isPreview ? 'server' : 'static',
   server: { port: localhostPort },
   site: siteUrl,
-  trailingSlash: 'always',
```

Comment:
This is required for the file proxy to work (or otherwise the file href need a `/` suffix like `dummy.pdf/`) which is odd. I'm okay with having the required trailing slash removed as all our routing and canonicals make sure we have consistent page routes ending with a `/`.

Reply:
Do we have tests in place to verify that trailing slashes for routes to records other than files do in fact work?

Reply:
Not yet. I see the routing tests only cover redirects: https://github.com/voorhoede/head-start/blob/main/src/lib/routing/index.test.ts#L31
Will add tests

Reply:
Added tests. See commit https://github.com/voorhoede/head-start/pull/205/commits/587caf9b56a9035154297db944c7e65d2f73c4c4

Reply:
Figured I should actually also include tests for the LinkToFile component itself. WIP: https://github.com/voorhoede/head-start/pull/205/commits/970401f4a1d099dcb4d377f287e48c00a75619af (didn't have enough time to finish it)